safe/gis/vector/test/test_intersection.py

```diff
@@ -26,21 +26,21 @@ def tearDown(self):
     def test_intersection_mask_vector(self):
         """Test we can intersect two layers, like exposure and aggregation."""
 
-        hazard = load_test_vector_layer(
-            'gisv4', 'hazard', 'classified_vector.geojson')
+        exposure = load_test_vector_layer(
+            'gisv4', 'exposure', 'roads.geojson')
 
         aggregation = load_test_vector_layer(
```

Comment:
I added a comment about this. The test takes an aggregation layer, but I give its a hazard layer. THanks for review